lsp/tailwindcss-language-server/dist/index.js

```diff
@@ -6905,7 +6905,9 @@ module.exports = (function(e, t) {
               );
             }
           } else {
-            return o.substr(t).indexOf(e[2]) === -1;
+            // TODO is this correct?
+            // return o.substr(t).indexOf(e[2]) === -1;
+            return true;
```

Comment:
@iamcco If you could take a look at this line when you get a chance, it'd be much appreciated.

Reply:
undo this chunk and add below code before this chunk.

```
          } else if (/%tw ['"]/.test(o.substr(t))) {
            return o.substr(t + 5).indexOf(e[2]) === -1;
```